test/mocha.opts

```diff
@@ -1 +0,0 @@
---timeout 60000
```

Comment:
What's the reason for deleting this? I put it in because in Travis, the tests sometimes wouldn't finish in the default time allotted.